src/main/java/org/generationcp/middleware/service/impl/study/StudyInstance.java

```diff
@@ -9,18 +9,23 @@
 public class StudyInstance {
 
 	private int instanceDbId;
+	private int experimentId;
```

Comment:
Question: why do we need the `experimentId `exposed?

Reply:
@darla-leafnode We need to return this to the front-end so that saving of Environment Details/Study Conditions values will work.